feature/sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/Search.kt

```diff
@@ -372,7 +372,8 @@ private fun SearchTextField(
                     bottom = 16.dp,
                     start = 0.dp,
                     end = 16.dp
-                )
+                ),
+                container = {},
```

Comment:
In [this commit](https://android.googlesource.com/platform/frameworks/support/+/b2c3164b907798ad87c70db5c2d35b0745f1948f), `FilledContainerBox` is introduced and be set as default container for `TextFieldDecorationBox`. (ref [link](https://android.googlesource.com/platform/frameworks/support/+/b2c3164b907798ad87c70db5c2d35b0745f1948f%5E%21/#F10))
As you can see at doc comment, "the container to be drawn behind the text field. By default, this includes the bottom indicator line."

It can be modified with setting indicator color on `TextFieldColors` (ref: [38578d0](https://github.com/DroidKaigi/conference-app-2022/pull/875/commits/38578d0f9c89241ab844d4610ff0cabd7e82a906)), but I think it is not need to set container here because indicatorLine is not necessary here.
so I'm passing a empty composable.